internal/scorecard/scorecard.go

```diff
@@ -195,13 +200,20 @@ func (r PodTestRunner) Cleanup(ctx context.Context) (err error) {
 	if err != nil {
 		return err
 	}
+
 	return nil
 }
 
 // RunTest executes a single test
 func (r PodTestRunner) RunTest(ctx context.Context, test v1alpha3.TestConfiguration) (*v1alpha3.TestStatus, error) {
+
 	// Create a Pod to run the test
 	podDef := getPodDefinition(r.configMapName, test, r)
+
+	if test.Storage.Spec.MountPath.Path != "" {
```

Comment:
I don't see the top-level [`v1alpha3.Configuration.Storage`](https://github.com/operator-framework/api/blob/master/pkg/apis/scorecard/v1alpha3/configuration_types.go#L24) field being respected here.

Reply:
Good catch, it looks like this is the first field on the scorecard config that needs to be propagated to individual tests